Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -783,6 +792,8 @@ void GameListCtrl::RescanList()
   // Post UI event to update the displayed list
   QueueEvent(new wxCommandEvent(DOLPHIN_EVT_REFRESH_GAMELIST));
 
+  post_status(_("Scan complete!"));
```

Comment:
It only stays there until something overwrites it, which is typically as soon as you start a game.